packaging/npm/package.json.in

```diff
@@ -33,9 +33,9 @@
   ],
   "logo": "https://s3.amazonaws.com/manifold.co/torus_logo.png",
   "license": "BSD-3-Clause",
-  "dependencies": {
+  "dependencies": {},
+  "devDependencies": {},
+  "optionalDependencies": {
```

Comment:
Interesting discussion about conditional dependencies: https://github.com/npm/npm/issues/7340

Reply:
Tested out using this to skip installing kexec entirely, stalls out on windows due to npm bug.